views/partials/header.ejs

```diff
@@ -0,0 +1,9 @@
+<head>
+  <meta charset="utf-8">
+  <title>Shawns website</title>
```

Comment:
Change it to CS club website. That was my mistake on copy and paste from my website

Reply:
Sure thing

Reply:
Ok, fixed.